index.html

```diff
@@ -6,14 +6,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portofolio</title>
     <link rel="stylesheet" href="style.css">
-    </head>
+</head>
 <body>
 <header class="header">
         <div class="name">
           <a  class="mylogo" href="#">MY logo</a>
         </div>
         <nav class="menu">
           <a href=""><img class="icon-menu" src="./image/logobuger.JPG" alt=""></a>
+          <a class="Portfolio-desc" href="">Portfolio</a>
+          <a class="About-desc" href="">About</a>
+          <a class="contact-desc" href="">Contact</a>
```

Comment:
Please fix the font-style, size, color, font family and make sure you match the provided Figma design
Your design:
<img width="320" alt="Screenshot 2022-12-16 at 9 59 51 PM" src="https://user-images.githubusercontent.com/25887253/208171116-5d27981b-f81f-4311-935c-3a5e36cb4ef4.png">

Figma template:
<img width="564" alt="Screenshot 2022-12-16 at 10 10 27 PM" src="https://user-images.githubusercontent.com/25887253/208171477-d7e00c4c-6b12-408e-b412-3259566a7bdb.png">